package.json

```diff
@@ -40,6 +40,7 @@
       "**/@radix-ui/**",
       "**/react",
       "**/react-map-gl",
+      "**/rollup",
```

Comment:
Just curious, why `nohoist` over `resolutions`?

Reply:
My thinking on this was that using resolutions would pin the version of rollup used to build ui and react packages to the version that vite.js needs. It seemed better to leave the rollup versions needed by each build alone. Thoughts?

Reply:
Makes sense to me 💯